net/respondd/src/respondd.c

```diff
@@ -649,10 +645,7 @@ int main(int argc, char **argv) {
 			break;
 
 		case 'd':
-			if (chdir(optarg)) {
```

Comment:
Doesn't really matter if it's the first or the last, whatever is nicer to implement...